src/developTest/java/mServer/crawler/sender/newsearch/ZdfDatenFilmTest.java

```diff
@@ -0,0 +1,40 @@
+package mServer.crawler.sender.newsearch;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+
+import de.mediathekview.mlib.Const;
+
+@RunWith(Parameterized.class)
+public class ZdfDatenFilmTest {
+    @Parameters
+    public static Collection<Object[]> data() {
+        return Arrays.asList(new Object[][] {   
+            { "my topic", "url", "zdfmy topicurl" },
+            { "topic1", "https://rodlzdf-and-so-on.de/12/11.mp4", "zdftopic1https://rodlzdf-and-so-on.de/12/11.mp4" },
+            { "topic2", "https://nrodlzdf-and-so-on.de/12/11.mp4", "zdftopic2https://rodlzdf-and-so-on.de/12/11.mp4" },
+            { "topic3", "http://rodlzdf-and-so-on.de/12/11.mp4", "zdftopic3http://rodlzdf-and-so-on.de/12/11.mp4" },
+            { "topic4", "http://nrodlzdf-and-so-on.de/12/11.mp4", "zdftopic4http://rodlzdf-and-so-on.de/12/11.mp4" },
+        });
+    }
+    
+    private final ZdfDatenFilm target;
+    private final String expectedIndex;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1213)